<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionWindow</class>
 <widget class="QDialog" name="ActionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>570</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>동작 편집</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: #EFEFEF;
}</string>
  </property>
  <widget class="QWidget" name="actionwindow">
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <widget class="QFrame" name="frame_main">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>480</width>
      <height>550</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;
border:1px solid black;
border-radius: 5px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>340</width>
       <height>340</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget {
    color: black;
    border: 1px solid #8f8f91;
    border-radius: 5px;
    background-color: white;
  }
  QTableWidget::item {
    border: none;
    padding: 4px;
  }
  QTableWidget::item:selected {
    background-color: #0052cc;
    color: white;
  }
  QScrollBar {
    width: 0px;
    height: 0px;
  }</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDropMode::DragOnly</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::DropAction::IgnoreAction</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QPushButton" name="button_add">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>80</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
       <hintingpreference>PreferVerticalHinting</hintingpreference>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: #EFEFEF;
    padding: 2px 4px;
}</string>
     </property>
     <property name="text">
      <string>동작 추가</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::InsertText"/>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_delete">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>320</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
       <hintingpreference>PreferVerticalHinting</hintingpreference>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: #EFEFEF;
    padding: 2px 4px;
}</string>
     </property>
     <property name="text">
      <string>삭제</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::InsertText"/>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_edit">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>260</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
       <hintingpreference>PreferVerticalHinting</hintingpreference>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: #EFEFEF;
    padding: 2px 4px;
}</string>
     </property>
     <property name="text">
      <string>수정</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::InsertText"/>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_delay">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>140</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
       <hintingpreference>PreferVerticalHinting</hintingpreference>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: #EFEFEF;
    padding: 2px 4px;
}</string>
     </property>
     <property name="text">
      <string>대기시간 추가</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::InsertText"/>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_save">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>510</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
       <hintingpreference>PreferVerticalHinting</hintingpreference>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: #EFEFEF;
    padding: 2px 4px;
}</string>
     </property>
     <property name="text">
      <string>저장</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::InsertText"/>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_cancel">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>510</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
       <hintingpreference>PreferVerticalHinting</hintingpreference>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: #EFEFEF;
    padding: 2px 4px;
}</string>
     </property>
     <property name="text">
      <string>취소</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::InsertText"/>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_down">
     <property name="geometry">
      <rect>
       <x>335</x>
       <y>435</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
       <hintingpreference>PreferVerticalHinting</hintingpreference>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-radius: 5px;
	border: 1px solid #8f8f91;
	background-color: #EFEFEF;
	color:black;
	padding: 2px 4px;
}</string>
     </property>
     <property name="text">
      <string>∨</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::InsertText"/>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_up">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>435</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
       <hintingpreference>PreferVerticalHinting</hintingpreference>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-radius: 5px;
	border: 1px solid #8f8f91;
	background-color: #EFEFEF;
	color:black;
	padding: 2px 4px;
}</string>
     </property>
     <property name="text">
      <string>∧</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::InsertText"/>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_title">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>1</y>
       <width>478</width>
       <height>60</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>128</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: deepskyblue;
color: white;
border: none;
border-top-left-radius :5px;
border-top-right-radius : 5px;
border-bottom-left-radius : 0px;
border-bottom-right-radius : 0px;</string>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>macro name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="button_priority">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>200</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
       <hintingpreference>PreferVerticalHinting</hintingpreference>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: #EFEFEF;
    padding: 2px 4px;
}</string>
     </property>
     <property name="text">
      <string>우선순위 지정</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::InsertText"/>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_setting">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>380</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
       <hintingpreference>PreferVerticalHinting</hintingpreference>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: #EFEFEF;
    padding: 2px 4px;
}</string>
     </property>
     <property name="text">
      <string>설정</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::InsertText"/>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_window">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>435</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
    border: 1px solid #8f8f91;
    border-radius: 3px;
    padding: 1px 3px 1px 3px;
    background: white;
    selection-color: white;
    selection-background-color: blue;
}</string>
     </property>
     <property name="placeholderText">
      <string>선택하세요</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_arrowup">
     <property name="geometry">
      <rect>
       <x>182</x>
       <y>95</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>88</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #8f8f91;
background: transparent;
padding: 2px;
border:none;</string>
     </property>
     <property name="text">
      <string>▲</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_arrowdown">
     <property name="geometry">
      <rect>
       <x>182</x>
       <y>390</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>88</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #8f8f91;
background: transparent;
padding: 2px;
border:none;</string>
     </property>
     <property name="text">
      <string>▼</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pgselect">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>435</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
color:black;</string>
     </property>
     <property name="text">
      <string>프로그램 선택 :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
